introduction.md

```diff
@@ -0,0 +1,2 @@
+Hello dojo rules. I am sripathichaithanya.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.

Reply:
It looks perfect now +1 accept the request.